### PR TITLE
Update LICENSE.md

### PR DESCRIPTION
**Pull request by ethanschen**
### Description of the Change
- Updating license year in LICENSE.md file
### Release Notes
- It's 2021.

-----
[View rendered LICENSE.md](https://github.com/karezachen/atom/blob/master/LICENSE.md)

### CODE REVIEW

**Review by ThatXliner — APPROVED** [review]
Looks good! I'm not a maintainer so I can't merge this, though.

**Review by sadick254 — APPROVED** [review]
LGTM

**Pull request merged**
